### PR TITLE
main page feeded from database

### PR DESCRIPTION
**Pull request by krzysiekpom**
Environment setup done (webpack+babel+express+react+mongo). Main page displays the data fetched from the database (depending on environment either locally or from Heroku/mLab). No display of cards yet (only list of decks).

- I didn't do any testing so far, I need to take a better look at testing in Node (there are two tests I made just to check the syntax but they're testing the framework)
- Let me know if you're seeing any crucial mistakes here (e.g. wrong models/components structure), I'm making progress here but for the most part still don't know what I'm doing

Thanks!

https://guarded-refuge-80701.herokuapp.com/

### CODE REVIEW

File: components/index.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render } from 'react-dom';
+import Main from './Main/Main';
+
+const _ = require('lodash/core');
```
Comment (polakowski) [inline components/index.jsx]: Why not just `import _ from 'lodash/core';`?

File: components/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      decks: []
+    };
+  }
+
+  componentDidMount() {
+    fetch('/api/decks')
```
Comment (polakowski) [inline components/index.jsx]: Good idea is to extract this functionality to module, and then use 
```jsx
Api.get('/decks').then((response) => handleResponse(response));
```
Without using chained `then` calls every time.

File: components/index.jsx (hunk 1 above)
Comment (krzysiekpom) [reply components/index.jsx]: Done. I moved it to module - though I'm not sure if all fetched data will need to be converted to array yet.

File: components/index.jsx (hunk 2 above)
Comment (krzysiekpom) [reply components/index.jsx]: Done. For now I replaced handleResponse with direct call on setState. I'm not sure what would be the best way to go here - if the only argument I pass to handleResponse is "res", then how does it know what variable should it assign "res" to? Should I make separate handler for each assignment (e.g. handleDeck)?

**Pull request merged**